framework/test/test_gen_tests.sh

```diff
@@ -60,7 +60,7 @@ init
 
 # Run all bugs, unless otherwise specified
 if [ "$BUGS" == "" ]; then
-    num_bugs=$(num_lines $BASE_DIR/framework/projects/$PID/commit-db)
+    num_bugs=$(num_lines $BASE_DIR/framework/projects/$PID/$BUGS_CSV_ACTIVE)
     BUGS="$(seq 1 1 $num_bugs)"
```

Comment:
This should be replaced with a call to get_bug_ids. This also makes the check on line 86 obsolete.

Reply:
Fixed